### PR TITLE
Feat: Updated Readmes

### PR DESCRIPTION
**Pull request by Meldiron**
Improved readmes, and moved some sections. This will help implement new runtimes faster without duplicating a lot of sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +52,45 @@ Runtime environments for serverless cloud computing for multiple coding language
 | Node.js | 16.0    | [open-runtimes/node.js:16.0](https://hub.docker.com/r/open-runtimes/node.js) | [Node.js Examples]() | [![Docker Pulls](https://img.shields.io/docker/pulls/open-runtimes/node.js?color=f02e65&style=flat-square)](https://hub.docker.com/r/open-runtimes/node.js) |
 | PHP     | 8.0     | [open-runtimes/php:8.0](https://hub.docker.com/r/open-runtimes/php)      | [PHP Examples]() | [![Docker Pulls](https://img.shields.io/docker/pulls/open-runtimes/php?color=f02e65&style=flat-square)](https://hub.docker.com/r/open-runtimes/php) |
 
+## Runtimes introduction
+
+All runtimes share the same folder structure, but each also adds runtime-specific files to properly support it's package manager.
+
```
Comment (christyjacob4) [inline README.md:58]: I recommend using the `tree ` command within a runtime folder to give a better representation

```
.
├── build.sh
├── docker-compose.yml
├── Dockerfile
├── example
├── example.sh
├── launch.sh
├── README.md
└── server.X

1 directory, 9 files
```

After this, you can have bullet points to explain the function of each file/ directory 

- `example/` - This folder contains a sample cloud function that you can use to test the runtime. 
- `Dockerfile` -  Defines how to build the runtime with instructions to prepare the environment for the runtime. These images are usually based on official Alpine or Ubuntu images.

... And so on for the other files and folders.

File: README.md [hunk 2]
```diff
@@ -51,6 +52,45 @@ Runtime environments for serverless cloud computing for multiple coding language
...
+The runtime has 3 main actions scripts:
+
+- Build source code into executable script. This action is described in `build.sh` and can be either package installations, or build process of the runtime.
+- Take executable script and prepare server that will execute it. File `launch.sh` explains this process, and runs HTTP server on port `3000`.
+- All-in-one `example.sh` script that takes content of `example` directory, builds it, and launched HTTP server. This script can be easily executed by running `docker-compose up` thanks to `docker-compose.yml` file.
+
+A runtime-specific HTTP server implementation can be found in `server.X` file where `X` is file extention used by specific programming language, for instance `server.js`.
+
+Every request sent to any of the runtimes must have header `X-Internal-Challenge`. The value of this header has to match the value of environment variable `INTERNAL_RUNTIME_KEY` set on the runtime. All example scripts use `example1234` key and we strongly recommend adjusting this key before production use.
```
Comment (christyjacob4) [inline README.md:71]:
```suggestion
Every request sent to any of the runtimes must have header `X-Internal-Challenge`. The value of this header has to match the value of environment variable `INTERNAL_RUNTIME_KEY` set on the runtime. All example scripts use `example1234` as the key and we strongly recommend adjusting this key before production use.
```

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Just some minor comments. Rest looks good

File: README.md (hunk 1 above)
Comment (Meldiron) [reply README.md:58]: Added folder structure

**Pull request merged**